internal/domain/transitions.go

```diff
@@ -762,7 +772,7 @@ func doTransition(transitionID uuid.UUID) {
 			comp.Task.State = model.TaskState_Sending
 			comp.Task.Operation = powerActionOp
 			trsTaskMap[trsTaskList[trsTaskIdx].GetID()] = comp
-			trsTaskList[trsTaskIdx].RetryPolicy.Retries = 3
+			trsTaskList[trsTaskIdx].CPolicy.Retry.Retries = 3
```

Comment:
Should this be set to the retries variable sent in?

Reply:
The global variable PCS_STATUS_HTTP_RETRIES is for status requests.  At some point we may also want one for power caps and transitions.